cmd/fs-v1.go

```diff
@@ -226,6 +226,21 @@ func (fs *FSObjects) StorageInfo(ctx context.Context, _ bool) StorageInfo {
 	return storageInfo
 }
 
+// GetOBDLock - returns a locked cluster-wide lock to prevent parallel OBD operations
+func (fs *FSObjects) GetOBDLock(ctx context.Context) (RWLocker, error) {
```

Comment:
This this is added to the interface, we might as well make it generic.
```suggestion
func (fs *FSObjects) GetRWLock(ctx context.Context, name string, timeout) (RWLocker, error) {
```

Reply:
:+1: